app/src/main/java/fr/free/nrw/commons/contributions/RxJava2Tasks.java

```diff
@@ -0,0 +1,30 @@
+package fr.free.nrw.commons.contributions;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Locale;
+
+import fr.free.nrw.commons.PageTitle;
+import io.reactivex.Single;
+
+class RxJava2Tasks {
+
+    private static final String UPLOAD_COUNT_URL_TEMPLATE =
+            "https://tools.wmflabs.org/urbanecmbot/uploadsbyuser/uploadsbyuser.py?user=%s";
+
+    static Single<Integer> getUploadCount(String userName) {
```

Comment:
Any chance of adding a unit test to aid in documenting what goes to and fro to the server?  The `ApacheHttpClientMediaWikiApiTest` has code for starting and stopping a mock server to enqueue responses for the Wikimedia endpoints, so shouldn't be too bad to pull something together.  (and a valid answer to this question is "no" 😉  )